src/components/todoList/TodoListForm.tsx

```diff
@@ -19,25 +19,25 @@ export function TodoForm() {
     },
   );
 
-  const handleChange = (e) => {
+  const handleChange = (e:any) => {
     const { value } = e.target;
-    const id = Date.now();
 
     setFormInputData({
       ...formInputData,
-      id,
+      id:Date.now().toString(),
       [e.target.name]: value,
     });
   };
 
-  const handleSubmit = (e) => {
+  const handleSubmit = (e:any) => {
     if (formInputData.inputGoal) {
-      const newData = (data) => ([...data, formInputData]);
+      const newData = (data:any) => ([...data, formInputData]);
```

Comment:
we can't define type as any. It's an array and I think it's possible to give its type. @pulkitvyas08 what kind of array does db return?

Reply:
> we can't define type as any. It's an array and I think it's possible to give its type. @pulkitvyas08 what kind of array does db return or accept?

There is no db for goals yet. When the db is created we would need to make an API accordingly and there we can define what it would return or accept

Reply:
Then I think `any` is actually a good fit here since the type is bound to change. Drop a _TODO:_ and change it in the future

Reply:
Plus using `any` as a type for a callback is ok since you can't invoke it anywhere else. It's a once used, thrown away construct